### PR TITLE
List all known SLD bonus cards in the product description

### PR DESCRIPTION
**Pull request by kodawah**
This offers more control and ease of updating, as well as an attempt at taking variance into consideration (where needed).

### CODE REVIEW

File: data/contents/SLD.yaml [hunk 1]
```diff
@@ -1127,146 +1127,520 @@ products:
...
+    - foil: true
+      name: Seraph Sanctuary
+      number: 733
+      set: sld
   Secret Lair Drop Artist Series Aleksi Briclot:
     card_count: 5
     deck:
     - name: Artist Series Aleksi Briclot
       set: sld
+    card:
```
Comment (axxroytovu) [inline data/contents/SLD.yaml:1284]: I think this card should be foil

File: data/contents/SLD.yaml [hunk 2]
```diff
@@ -1127,146 +1127,520 @@ products:
...
+    - name: Fleshbag Marauder
+      number: 725
+      set: sld
   Secret Lair Drop Artist Series Kev Walker Foil:
-    card_count: 4
+    card_count: 5
     deck:
     - name: Artist Series Kev Walker foil
       set: sld
+    - foil: true
```
Comment (axxroytovu) [inline data/contents/SLD.yaml:1442]: Missing the `card:` line

Comment (kodawah) [reply data/contents/SLD.yaml:1442]: ✅

File: data/contents/SLD.yaml (hunk 1 above)
Comment (kodawah) [reply data/contents/SLD.yaml:1284]: actually no, Thought-Knot Seer is foil/nonfoil so it'd make sense that the nonfoil is found in the nonfoil bundle

**Comment by axxroytovu** [comment]
I salute you for working on this o7

**Comment by kodawah** [comment]
I think this is ready for review, there is a bunch of updates and fixes (notably viscera seer only coming from one drop)

the main missing groups are the drops with the jumpstart lands and slivers, I started filling a few but then realized it was getting messy and inaccurate so I'd rather start with these

here are some notes from taw/mtgwiki of the missing sets if anybody has more information
[sld notes.txt](https://github.com/user-attachments/files/16114478/sld.notes.txt)

**Comment by kodawah** [comment]
oh the checks are failing because the boosters are not loaded in yet

File: data/contents/SLD.yaml [hunk 3]
```diff
@@ -1733,26 +2651,47 @@ products:
...
+      name: Dakkon Blackblade
+      number: 698
+      set: sld
   Secret Lair Drop Full Sleeves The Tattoo Pack:
     card_count: 6
     deck:
     - name: Full Sleeves The Tattoo Pack
       set: sld
+    pack:
+    - name: stainedglass-tattoo
```
Comment (axxroytovu) [inline data/contents/SLD.yaml:2693]: This is breaking because it should be `code` not `name`

File: data/contents/SLD.yaml [hunk 4]
```diff
@@ -1833,6 +2792,9 @@ products:
     deck:
     - name: International Womens Day foil
       set: sld
+    pack:
+    - name: stainedglass-iwd
```
Comment (axxroytovu) [inline data/contents/SLD.yaml:2796]: Same here - should be `code` and not `name`

**Review by axxroytovu — CHANGES_REQUESTED** [review]

File: data/contents/SLD.yaml (hunk 3 above)
Comment (kodawah) [reply data/contents/SLD.yaml]: thanks for spotting, updated!

File: data/contents/SLD.yaml (hunk 4 above)
Comment (kodawah) [reply data/contents/SLD.yaml]: fixed too

**Review by axxroytovu — APPROVED** [review]

**Pull request merged**